frequently-asked-questions.md

```diff
@@ -0,0 +1,94 @@
+# Frequently Asked Questions (FAQ)
+
+This is a list of the most frequently asked questions by members of this community.
+
+## Engaging with Community
+
+**Q: I am new to open source. Where should I start contributing?**
+A: If it's your first time contributing to a project, please build and explore [AnitaB.org Open Source](https://github.com/anitab-org). Some repositories have some "first timers only" issues for newcomers. For example, Mentorship Backend contains [a list of “first timers only” issues](https://github.com/anitab-org/mentorship-backend/issues?q=is%3Aissue+is%3Aopen+label%3A%22First+Timers+Only%22) regarding both Documentation and Quality Assurance.
+
+**Q: What is and who can join Community Open Sessions?**
+A: These are sessions where anyone interested in discussing community concerns or projects can join regardless of title, membership level, repository access, or current program involvement.
+
+**Q: What is the minimal technical knowledge needed to contribute to AnitaB.org projects?**
+A: **Zero technical knowledge needed!** Because you can always help with documentation, testing the app, and creating wireframes to update it to the new branding guidelines.
+i.e. Reaching out to a therapist for PowerUp scenario content, reviewed UI designs for Wheelchairs from the Cerebral Palsy communities on Facebook.
```

Comment:
Nice question, I think this came from the time we were maintaining the [PowerUp project](https://github.com/anitab-org/powerup-android), in which outreach examples of contributions were "obvious". Because as PowerUp was a game scenario type of app, to build the scenarios for pre adolescents, we could reach out to people with experience to ask for help building appropriate content. Now that we don't maintain that project, we could have other examples 🤔 I will take that example out, and then we can add as we see fit. @vj-codes